bigframes/core/blocks.py

```diff
@@ -1192,13 +1192,23 @@ def pivot(
         *,
         columns: Sequence[str],
         values: Sequence[str],
+        columns_unique_values: typing.Optional[
```

Comment:
Can we document this parameter or pick a different name? It's not super intuitive to me what the purpose is without having the context from our design review about avoiding the query for the pivot operation.

Reply:
Added comment, PTAL